automated/linux/docker-prune/prune-lib.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+verify_app_present () {
+    app="$1"
+
+    if docker ps | grep -q "${app}"; then
+        echo "${app} container found"
+        return 0
+    else
+        echo "${app} container not found"
+        return 1
+    fi
+}
+
+check_image () {
+    image_name="$1"
+
+    if [[ "$(docker images -q "${image_name}" 2> /dev/null)" == "" ]] ; then
+        echo "Docker image '${image_name}' not found."
+        return 1
+    else
+        echo "Docker image '${image_name}' found."
+        return 0
+    fi
+}
+
+get_image_sha () {
+    ls /var/sota/reset-apps/apps/shellhttpd > "$(pwd)/sha.txt"
+}
+
+check_image_prune () {
+    runtime="5 minutes"
```

Comment:
Does it make sense to have this as a variable that we can change from the yaml file?

Reply:
Well, it's only really there as a maximum time to run before breaking the loop. In theory, it should never take that long unless there is a problem.

Reply:
OK